src/object/thunderstorm.cpp

```diff
@@ -30,9 +32,8 @@
 namespace {
 
 const float LIGHTNING_DELAY = 2.0f;
-const float FLASH_DISPLAY_TIME = 0.1f;
+const float FLASH_DISPLAY_TIME = 2.0f;
```

Comment:
two seconds? What did this mean again?

Reply:
flash_display_timer; /**< counts down while flash is displayed */ So, yeah, that is the time that the flash is displayed.